feature/repair/src/main/java/com/iteneum/repair/data/RepairViewModel.kt

```diff
@@ -0,0 +1,80 @@
+package com.iteneum.repair.data
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.core.net.toUri
+import androidx.core.text.isDigitsOnly
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.RepairRequest
+import com.iteneum.UIEventRepair
+import com.iteneum.network.DataState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * File that contains [RepairViewModel] that works as ViewModel for the Repair View.
+ * It will handle the data work & process when loading & be done with the view.
+ *
+ * @author Jose Miguel Garcia Reyes
+ * @author Carlos Andres Perez Hernandez
+ */
+
+@HiltViewModel
+class RepairViewModel @Inject constructor() : ViewModel() {
+    var repairRequest: RepairRequest by mutableStateOf(RepairRequest())
+        private set
+
+    fun getInformation() = viewModelScope.launch {
+        when (response) {
```

Comment:
line 31, add remining states for response, acctually is just success state, you should add error state and loading state